src/test/java/org/mule/extensions/jms/test/util/ActiveMQConnectionFactoryUtilTestCase.java

```diff
@@ -18,6 +20,23 @@ public void testVersionValid() {
     assertFalse(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.14.5"));
     assertFalse(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("4.10.5"));
     assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.15.6"));
+    assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.18.3"));
     assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("6.0.0"));
   }
+
+  @Test
+  public void testUrlFormatWithQueryParameters() {
+    assertEquals(null,
+                 "failover:(ssl://localhost:61616?socket.verifyHostName=false,ssl://localhost:61616?socket.verifyHostName=false)?maxReconnectAttempts=5&useExponentialBackOff=false&randomize=false&jms.prefetchPolicy.all=5",
+                 ActiveMQConnectionFactoryUtil.brokerUrlFormat(
+                                                         "failover:(ssl://localhost:61616,ssl://localhost:61616)?maxReconnectAttempts=5&useExponentialBackOff=false&randomize=false&jms.prefetchPolicy.all=5",
+                                                         false));
+  }
+
+  @Test
+  public void testUrlFormatWithoutQueryParameters() {
+    assertEquals(null,
+                 "failover:(ssl://localhost:61616?socket.verifyHostName=false,ssl://localhost:61616?socket.verifyHostName=false)",
+                 ActiveMQConnectionFactoryUtil.brokerUrlFormat("failover:(ssl://localhost:61616,ssl://localhost:61616)", false));
+  }
```

Comment:
Shouln't we add a test to verify the behavior when the verifyHostName parameter is true? 
What would it happen if I tests the following scenarios?:

- ActiveMQConnectionFactoryUtil.brokerUrlFormat(brokerURL, `true`)
- ActiveMQConnectionFactoryUtil.brokerUrlFormat("", `true`)
- ActiveMQConnectionFactoryUtil.brokerUrlFormat(null, `true`)
- "failover:(ssl://localhost:61616?socket.verifyHostName=false,ssl://localhost:61616)?maxReconnectAttempts=5"
- "failover:( ssl://localhost:61616,   ssl://localhost:61616  )?maxReconnectAttempts=5"
- "failover:(ssl://localhost:61616)?maxReconnectAttempts=5"
- "failover:( invalidURL  )?maxReconnectAttempts=5"

Reply:
done